python/web/index.py

```diff
@@ -0,0 +1,139 @@
+import os
+import flask
+import gevent.pywsgi
+# import pymysql
+
+import pshexs
+
+app = flask.Flask(__name__)
+
+
+@app.route('/python/')
+def root():
+    # return flask.render_template('base.html')
+    return create_list()
+
+
+def file_get_contents(example):
+    current_example = os.getcwd() + '/pshexs/{0}.py'.format(example)
+
+    text = open(current_example, "r")
+    contents = text.read()
+    text.close()
+    # resp = flask.make_response(contents)
+    # resp.headers['Content-Type'] = 'text/plain'
+    # return resp
+    return contents
+
+
+@app.route('/python/<example>')
+def add_example_route(example):
+
+    if hasattr(pshexs, example):
+        # current_example = os.getcwd() + '/pshexs/{0}.py'.format(example)
+        #
+        # text = open(current_example, "r")
+        # contents = text.read()
+        # text.close()
+        contents = file_get_contents(example)
+        resp = flask.make_response(contents)
+        resp.headers['Content-Type'] = 'text/plain'
+        return resp
+        # return file_get_contents(example)
+    else:
+        return "Sorry, no sample code is available."
+
+
+@app.route('/python/<example>/output')
+def add_example_output_route(example):
+
+    if hasattr(pshexs, example):
+        contents = getattr(getattr(pshexs, example), 'test_output')()
+
+        resp = flask.make_response(contents)
+        resp.headers['Content-Type'] = 'text/plain'
+        return resp
+    else:
+        return "Sorry, no sample code is available."
+
+
+def create_list():
+    names = {
+        "elasticsearch": "Elasticsearch",
+        "influxdb": "InfluxDB",
+        "memcached": "Memcached",
+        "mongodb": "MongoDB",
+        "mysql": "MySQL",
+        "postgresql": "PostgreSQL",
+        "rabbitmq": "RabbitMQ",
+        "redis": "Redis",
+        "solr": "Solr",
+    }
+    header = '<html>' \
```

Comment:
Triple-quoted strings will work way better here, methinks. That's more readable and maintainable.